thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -545,7 +545,8 @@ class CKeyedJoinSlave : public CSlaveActivity, public CThorDataLink, implements
     OwnedConstThorRow defaultRight;
     unsigned portbase, node;
     IArrayOf<IDelayedFile> fetchFiles;
-    FPosTableEntry *fPosToNodeMap; // maps fpos->node for all parts of logical file
+    FPosTableEntry *_fPosToNodeMap; // maps fpos->node for all parts of logical file
```

Comment:
It isn't clear what the difference between these two variables is - and a difference of a leading _ isn't very helpful.